src/plugins/console/public/src/kb.js

```diff
@@ -249,8 +249,8 @@ es.addServerChangeListener(function () {
   else if (version[0] === "2") {
     api = "es_2_0";
   }
-  else if (version[0] === "3") {
-    api = "es_2_0"; // TODO: change :)
+  else if (version[0] === "5") {
```

Comment:
nitpicky, but perhaps we could write this whole if thing simpler?

use a variable for `version[0]`, and remove the repeat of `"es_1_0"` three times.

No is a perfectly acceptable answer :).